integration-tests/src/main/java/io/quarkiverse/univocityparsers/it/UnivocityParsersResource.java

```diff
@@ -0,0 +1,106 @@
+/*
```

Comment:
No need for the license header

Reply:
@gastaldi  yes thank you, I remove it (Intellij hidden the header to me ...)